### PR TITLE
Add ConfirmInput interface specification

### PR DESCRIPTION
**Pull request by BigRoma**

### CODE REVIEW

**Review by sertony — APPROVED** [review]

File: ConfirmInput/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Interface for confirm input.
+
+## Functions
+
+`select` - prints prompt message to the user and returns true or false choice
+
+arguments:
+
+	handlerId: uint32
+    prompt: bytes
```
Comment (Keshoid) [inline ConfirmInput/README.md]:
```suggestion
	answerId: uint32 - id of function callback.
	prompt: bytes - utf-8 string to print to the user before input.
```

Comment (Keshoid) [reply ConfirmInput/README.md]: [confirm_input.tar.gz](https://github.com/tonlabs/DeBot-IS-consortium/files/5981436/confirm_input.tar.gz)

ConfirmInput.sol
ConfirmInput.abi.json
ConfirmInput.hpp
Example.sol

File: ConfirmInput/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+returns:
+
+	{ value: bool }
+
+## Declaration in Solidity
+
+TODO: Add declaration
```
Comment (Keshoid) [inline ConfirmInput/README.md]:
```suggestion
interface IConfirmInput {

	function select(uint32 answerId, string prompt) external returns (bool value);

}
```

File: ConfirmInput/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Declaration in C++
+
+TODO: Add declaration
```
Comment (Keshoid) [inline ConfirmInput/README.md]:
```suggestion
namespace tvm { namespace schema {

__interface IConfirmInput {
	[[internal, answer_id]]
	bool_t select(string prompt);
}

}};
```

File: ConfirmInput/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Code Example
+
+### Solidity
+
+TODO: Add example
```
Comment (Keshoid) [inline ConfirmInput/README.md]:
```suggestion
Example.sol
```

**Review by Keshoid — CHANGES_REQUESTED** [review]

**Review by Keshoid — APPROVED** [review]

**Review by sertony — APPROVED** [review]

**Review by mikhailUshakoff — APPROVED** [review]

File: ConfirmInput/ConfirmInput.sol [hunk 5]
```diff
@@ -0,0 +1,25 @@
+pragma ton-solidity >=0.35.0;
+
+interface IConfirmInput {
+
+	function select(uint32 answerId, string prompt) external returns (bool value);
```
Comment (mikhailUshakoff) [inline ConfirmInput/ConfirmInput.sol:5]: rename to get

**Review by azhogin — APPROVED** [review]

**Pull request merged**